File: core/src/com/broll/gainea/server/core/cards/impl/play/C_SuspendPlayer.java
package com.broll.gainea.server.core.cards.impl.play;

import com.broll.gainea.server.core.cards.AbstractCard;

public class C_SuspendPlayer extends AbstractCard {
    public C_SuspendPlayer() {
        super(12,"In den Kerker", "Ein Spieler deiner Wahl muss eine Runde aussetzen");
    }

    @Override
    public boolean isPlayable() {
        return true;
    }

    @Override
    protected void play() {
        selectHandler.selectOtherPlayer(owner, "Spieler der Aussetzen muss:").skipRounds(1);
    }
}

File: server/src/main/com/broll/gainea/server/core/cards/impl/play/C_SuspendPlayer.kt
package com.broll.gainea.server.core.cards.impl.play

import com.broll.gainea.server.core.cards.Card
import com.broll.gainea.server.core.cards.EffectType.OTHER

class C_SuspendPlayer :
    Card(12, OTHER, "In den Kerker", "Ein Spieler deiner Wahl muss eine Runde aussetzen") {
    init {
        drawChance = 0.4f
    }

    override val isPlayable: Boolean
        get() = true

    override fun play() {
        selectHandler.selectOtherPlayer(owner, "Spieler der Aussetzen muss:").skipRounds(1)
    }
}
